src/lib/db/index.ts

```diff
@@ -27,19 +27,25 @@ export interface AttachmentRow {
 export interface TransactionRow {
   id: UUID
   state: TransactionState
-  localId: UUID
+  local_id: UUID
```

Comment:
why are we mixing cases here?

Reply:
I believe we got this covered via slack, changing other to be snake case